node_lambnik/src/tiler/src/util/message-tile.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Creates an image with nothing but an error description on it
+ * so that users can actually receive error messages
+ */
+import Jimp from 'jimp'
+
+const createBlankImage = () => new Promise((resolve, reject) => {
+    // eslint yells at me for using 'new' for side effects
+    // (which I agree with) but this is how it's implemented
+    /* eslint-disable-next-line no-new */
+    new Jimp(256, 256, (err, image) => {
+        if (err) reject(err)
+        else resolve(image)
+    })
+})
+
+const toBuffer = image => new Promise((resolve, reject) => {
+    image.getBuffer(Jimp.MIME_PNG, (err, result) => {
+        if (err) reject(err)
+        else resolve(result)
+    })
+})
+
+export default message => createBlankImage()
+    .then(tile => Jimp.loadFont(Jimp.FONT_SANS_16_BLACK)
```

Comment:
A transparent image with black text is invisible on a black background.  If it's possible to put an outline around the text or a background around just the text, not the whole tile, that might be better than a whole-tile background.  But a whole-tile background would still, I think, be better than black text on an unknown (possibly also black or dark) background.